### ISSUE TITLE
[Feat]: Add a like or upvote feature

### ISSUE THREAD

**Issue by suzy-g38** [GrabBits/GrabBits_Website#129]
**Is your feature request related to a problem? Please describe.**

In the opportunities section, if we add a like/ upvote or a button in the hackathon which will show how many persons are interested, that can be a great way for the users to get interested in a certain hackathon. 

**Describe the solution you'd like**

 As there is a backend in this project we can use that to save those data.

**Describe alternatives you've considered**

**Additional context**

I would like to work on it under GSSOC'23

**Comment by Bhushan8177** [comment]
Great @suzy-g38 , Assigning to you , best of luck!!

**Comment by suzy-g38** [comment]
@Bhushan8177 I can use the databse right?

**Comment by Bhushan8177** [comment]
> @Bhushan8177 I can use the databse right?

Yes sure, you can ask for any of the credentials required on the discord, kindly ping me on discord!

**Comment by suzy-g38** [comment]
> > @Bhushan8177 I can use the databse right?
> 
> Yes sure, you can ask for any of the credentials required on the discord, kindly ping me on discord!

@Bhushan8177 Discord group or Discord DM?

**Comment by Bhushan8177** [comment]
discord dm !!
